spec/requests/carts_spec.rb

```diff
@@ -0,0 +1,94 @@
+require "rails_helper"
+
+RSpec.describe CartsController, type: :controller do
+  subject { assigns(:cart_items) }
+
+  let(:cart_items) { { "1" => 1, "2" => 1, "3" => 1 }.to_json }
+  let!(:product1) { FactoryBot.create(:product, id: 1, status: "enabled") }
```

Comment:
```suggestion
  let!(:product_1) { FactoryBot.create(:product, id: 1, status: "enabled") }
```
cho nó chuẩn chỉ nha

Reply:
dạ oke anh để em update lại